generator/src/main/java/com/linkedin/pegasus/generator/FileFormatDataSchemaParser.java

```diff
@@ -62,10 +63,15 @@ public FileFormatDataSchemaParser(String resolverPath, DataSchemaResolver schema
     _schemaParserFactory = schemaParserFactory;
   }
 
-  public DataSchemaParser.ParseResult parseSources(String sources[]) throws IOException
+  public DataSchemaParser.ParseResult parseSources(String[] sources) throws IOException
   {
     final DataSchemaParser.ParseResult result = new DataSchemaParser.ParseResult();
+    parseSources(sources, result);
+    return result;
+  }
 
+  void parseSources(String[] sources, DataSchemaParser.ParseResult result) throws IOException
```

Comment:
Should this be `public` since it is meant to be used by others

Reply:
I think it is only used in DataSchemaParser, so package private looks like a right approach to me.
@BrianPin why do you think we need to make a new API of FileFormatDataSchemaParser?

Reply:
+1 on Yingjie's comment. It's not meant for public consumption, at least not yet. Keeping things initially restricted makes it easier to make changes later.